package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0-alpha.0",
   "description": "",
   "scripts": {
-    "test": "jest --coverage",
+    "test": "node ./scripts/updateCompliationTests.js && jest --coverage",
```

Comment:
If I clone this repo and run `npm run test` will that run? 
I guess I need some Ruby setup on my machine don't I? 
If so, maybe this should be in his own script and only whoever has this set up will choose to run it

Reply:
No you will not need a local setup.

The script is smart enough to detect if you have the required environment variables, if you don't have them, we just skip this script completely.

This means that Travis, or any other person who has no idea this script exists (and has no ruby) can peacefully run their tests.

If however, a developer has everything set-up, this will generate an updated compilation test and run the tests on that file( as well as the other project files).